e2e/controller_test.go

```diff
@@ -0,0 +1,36 @@
+package e2e
+
+import (
+	"encoding/json"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+)
+
+func testController() {
+	var cc CleanupContext
+	BeforeEach(func() {
+		cc = commonBeforeEach()
+	})
+	AfterEach(func() {
+		commonAfterEach(cc)
+	})
+
+	It("should expose Prometheus metrics", func() {
+		stdout, stderr, err := kubectl("get", "pods", "-n=topolvm-system", "-l=app.kubernetes.io/component=controller,app.kubernetes.io/name=topolvm", "-o=json")
+		Expect(err).ShouldNot(HaveOccurred(), "stdout=%s, stderr=%s", stdout, stderr)
+
+		var pods corev1.PodList
+		err = json.Unmarshal(stdout, &pods)
+		Expect(err).ShouldNot(HaveOccurred())
+
+		pod := pods.Items[0]
+
+		// For CSI sidecar metrics access checking
+		for _, port := range []string{"9808", "9809", "9810", "9811"} {
+			stdout, stderr, err = kubectl("exec", "-n", "topolvm-system", pod.Name, "-c=topolvm-controller", "--", "curl", "http://localhost:"+port+"/metrics")
+			Expect(err).ShouldNot(HaveOccurred(), "stdout=%s, stderr=%s", stdout, stderr)
```

Comment:
This can be simplified as `kubectl exec -n topolvm-system deploy/topolvm-controller -c topolvm-controller -- curl http://...`.
Could you revise it?

Reply:
Thank you. I fixed it.